cypress/support/login/index.js

```diff
@@ -26,10 +26,10 @@ Cypress.Commands.add("loginUser", (email, password) => {
     .get(emailAddressInput)
     .type(email)
     .get(emailPasswordInput)
-    .type(password)
+    .type(password, { log: false })
     .get(signInButton)
     .click()
-    .get(allertPopupMessage, { timeout: 3000 })
+    .get(allertPopupMessage, { timeout: 30000 })
```

Comment:
Is `allert` a typo?

Reply:
Can we set up global timeout? Imo if we wait for server response more than 20sec then there is something not right.

Reply:
The alert is for "You are now logged in" message, you are right there is a typo.

Reply:
Timeouts removed, typo fixed.